BlockExplorer.vue

```diff
@@ -25,7 +25,7 @@
  */
 'use strict';
 
-import axios from 'axios';
+import {httpClient} from '@digitalbazaar/http-client';
 import Blocks from './Blocks.vue';
 
 const headers = {Accept: 'application/ld+json, application/json'};
```

Comment:
This can be removed since it's already being set here https://github.com/digitalbazaar/http-client/blob/main/main.js#L10